### DISCUSSION TITLE
[Website] PDF Brochure download isn't working?

### DISCUSSION THREAD

**Discussion by PureKrome** [ravendb/ravendb#13344] [General]
👋🏻 G'Day HR krew,

I think the icon to click the PDF brochure isn't working? When I click on it, it opens up a new tab .. and it's the same page? nothing downloaded?

![image](https://user-images.githubusercontent.com/899878/147113005-5ecdcd42-a6f1-4191-96d8-0e8bf2e43d68.png)

**Comment by mateuszbartosik** [comment]
Hi @PureKrome,
sorry for the trouble. It was just fixed. Can you please confirm it's working for you now?

**Reply by PureKrome** [reply]
This is embarrassing, but .. where is found on the website? I found it via the email newsletter. But I can't easily find it via the top-bar navigation?

**Reply by ayende** [reply]
It is here: https://ravendb.net/promotional-materials?

**Reply by PureKrome** [reply]
Ta - but where is this link on the website? can anyone post a pic please?

**Reply by ayende** [reply]
It's at the bottom of the page, see:

![image](https://user-images.githubusercontent.com/116915/147729311-cf9bc646-de67-45db-b498-d6c2872cd4f3.png)

**Comment by PureKrome** [comment]
AH! perfect! Side note - I didn't really notice that footer section. but yes, that's great. ta!
